lib/sorcery/model/temporary_token.rb

```diff
@@ -7,12 +7,14 @@ module Model
     # such as reseting password and activating the user by email.
     module TemporaryToken
       def self.included(base)
+        # FIXME: This may not be the ideal way of passing sorcery_config to generate_random_token.
+        @sorcery_config = base.sorcery_config
```

Comment:
Mildly worried that this might cause `@sorcery_config` to be unintentionally exposed, but not familiar enough with how ruby inheritance/privacy works to be sure. This is how it gets saved in `lib/sorcery/model.rb` however.